mix/CodeModel.h

```diff
@@ -64,6 +64,29 @@ public slots:
 	CodeModel* m_model;
 };
 
+using LocationPair = QPair<int, int>;
+
+struct SolidityType
```

Comment:
Solidity types hierarchy is too complex and does not directly map to what we handle in Mix. Mix only cares about how to present a value of a type to the user, so this is more like type formatting info. It is not that much work to keep it in sync since the code to support new types would need to be added anyway.